spec/expressions.md

```diff
@@ -4821,6 +4821,7 @@ A constant expression must be the `null` literal or a value with one of  the fol
 *  References to `const` parameters or local variables
 *  Parenthesized sub-expressions, which are themselves constant expressions.
 *  Cast expressions, provided the target type is one of the types listed above.
+*  `null` literal conversions
```

Comment:
The name of the null literal conversion does not use a code font. I think this would be

> * Null literal conversions

Reply:
Thank you, @gafter . I updated the line.